### PR TITLE
avoid warning in gravity point postprocessor

### PR DESCRIPTION
**Pull request by tjhei**
I am getting warnings about un-initialized variables for the dim=2
instantiation that doesn't work anyways. Fix this.

### CODE REVIEW

File: source/postprocess/gravity_point_values.cc [hunk 1]
```diff
@@ -52,10 +52,22 @@ namespace aspect
...
+      AssertThrow(false, ExcNotImplemented());
+      return std::pair<std::string,std::string>();
+    }
+
+
+    template <>
+    std::pair<std::string,std::string>
+    GravityPointValues<3>::execute (TableHandler &)
+    {
+      const int dim = 3;
```
Comment (gassmoeller) [inline source/postprocess/gravity_point_values.cc]: `const unsigned int`

File: source/postprocess/gravity_point_values.cc [hunk 2]
```diff
@@ -52,10 +52,22 @@ namespace aspect
       output_file_number (numbers::invalid_unsigned_int)
     {}
 
+
```
Comment (gassmoeller) [inline source/postprocess/gravity_point_values.cc]: I think we usually have 3 empty lines between functions (here and below).

File: source/postprocess/gravity_point_values.cc (hunk 1 above)
Comment (tjhei) [reply source/postprocess/gravity_point_values.cc]: Interesting point. We normally use unsigned ints, but our template parameters are always
```
template <int dim>
```
It looks like we mostly use signed ints in ASPECT in this situation, but ~75% unsigned inside deal.II:
```
$ grep "const int dim" -r deal-git/source/ | wc -l
25
$ grep "const unsigned int dim" -r deal-git/source/ | wc -l
87
```
I will summon @bangerth for his opinion. :-)

**Comment by gassmoeller** [comment]
Ok, lets not delay this PR further because of a not crucial discussion about `int` vs `unsigned int`. The content of the PR is definitely correct. We can think separately about unifying the templates if we ever want to. Thanks for the fix.

**Pull request merged**